.circleci/config.yml

```diff
@@ -10,7 +9,7 @@ commands:
     steps:
       - run:
           name: Clone bichard7-next-core
-          command: git clone --depth 1 https://github.com/ministryofjustice/bichard7-next-core.git ~/bichard7-next-core
+          command: git clone -b rick/update-bichard-process-human-task --depth 1 https://github.com/ministryofjustice/bichard7-next-core.git ~/bichard7-next-core
```

Comment:
Will be reverted before merging to main. Waiting on https://github.com/ministryofjustice/bichard7-next-core/pull/515 to be merged